package.json

```diff
@@ -69,7 +69,7 @@
     "https-proxy-agent": "^5.0.1",
     "is": "^3.3.0",
     "is-ip": "^3.1.0",
-    "isolated-vm": "4.4.2",
+    "isolated-vm": "4.5.0",
```

Comment:
Also why are we updating this? Did we test this?

Reply:
The older version of the package is not compatible with Node v18. This version fixes that issue.

Reply:
Got it (lets make sure we specifically test scenarios for User transform in such case)